src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
+        System.out.println("Excavator started its work ");;
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Excavator stopped working ");;
```

Comment:
The message in `System.out.println` should be more informative and indicate that the `Excavator` stopped working. For example: `System.out.println("Excavator has stopped working");`